test/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests/OpenApiWriterFactoryTests.cs

```diff
@@ -0,0 +1,42 @@
+using Microsoft.OpenApi;
+using Microsoft.OpenApi.Writers;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.IO;
+using System;
+
+namespace Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests
+{
+    [TestClass]
+    public class OpenApiWriterFactoryTests
+    {
+        [TestMethod]
+        public void CreateInstance_Should_Return_Json_Writer()
+        {
+            // Arrange
+            OpenApiFormat format = OpenApiFormat.Json;
+            StringWriter writer = new StringWriter();
```

Comment:
Please change this variable to the class private variable, and you can initialize it in the ["TestInitialize"].